web/locale/en_US/messages.po

```diff
@@ -2642,3 +2642,6 @@ msgstr "suggest features"
 
 msgid "to help make it better."
 msgstr "to help make it better."
+
+msgid "No dojo"
+msgstr "You don't have any Dojos - click here to start a dojo!"
```

Comment:
The msgid string is always the same as the msgstr in the English file. The msgid will be English in every po file and the msgstr is the corresponding translation (which is just the same for English in this case).